### DISCUSSION TITLE
LDM Notes for July 13th, 2022

### DISCUSSION THREAD

**Discussion by 333fred** [dotnet/csharplang#6283] [LDM Notes]
https://github.com/dotnet/csharplang/blob/main/meetings/2022/LDM-2022-07-13.md

## Agenda

1. Lambda default parameters

Notes for the 11th are still forthcoming, we're figuring out how we want to publish them.

**Comment by MgSam** [comment]
Do the lambda rules adopted allow this?

```C#
delegate void D(int i);
D d3 = (int i = 0) => {}; // ??
```

**Reply by CyrusNajmabadi** [reply]
We have currently said that that would be an error since it's unclear in what scenario this would be useful.
